middleware/envvar/envvar_test.go

```diff
@@ -1,27 +1,26 @@
+//nolint:bodyclose // Much easier to just ignore memory leaks in tests
 package envvar
 
 import (
+	"context"
 	"encoding/json"
 	"io"
 	"net/http"
-	"os"
 	"testing"
 
 	"github.com/gofiber/fiber/v2"
 	"github.com/gofiber/fiber/v2/utils"
 )
 
 func TestEnvVarStructWithExportVarsExcludeVars(t *testing.T) {
-	os.Setenv("testKey", "testEnvValue")
-	os.Setenv("anotherEnvKey", "anotherEnvVal")
-	os.Setenv("excludeKey", "excludeEnvValue")
-	defer os.Unsetenv("testKey")
-	defer os.Unsetenv("anotherEnvKey")
-	defer os.Unsetenv("excludeKey")
+	t.Setenv("testKey", "testEnvValue")
```

Comment:
This might need to be reverted. Do we really want to continue supporting Go 1.16? Support ended nearly a year ago and security issues have since been found.

Reply:
I think we should drop 1.16 support with 1.19. What do you think @ReneWerner87